src/fontra/views/editor/editor.js

```diff
@@ -643,24 +664,74 @@ export class EditorController {
     });
   }
 
-  addEditTool(tool) {
+  addEditTool(tool, wrapper_id = "edit-tools") {
```

Comment:
Can you describe briefly what the `wrapper_id` does and why it is needed?

Separate from that: following Fontra naming conventions it should be called `wrapperID`.

Reply:
I wrap all sub tools into one div here, so that I can eg stack them visually and do further modifications like moving the child to first place when selected etc.
https://github.com/googlefonts/fontra/blob/0ca296368f68528b0603554413d5cce88b041749/src/fontra/views/editor/editor.js#L624

Reply:
Renamed wrapper_id to wrapperID

Reply:
I make this as resolved as we did a coding session together.